app/src/main/java/com/protify/protifyapp/features/events/EventDetails.kt

```diff
@@ -60,16 +68,40 @@ class EventDetails {
                     )
                 }
             }
-            FloatingActionButton(
-                onClick = {
-                    navController.navigate("editEvent/${date}/${eventId}")
-                },
+            Row(
                 modifier = Modifier
                     .align(Alignment.BottomEnd)
-                    .padding(24.dp),
-                containerColor = MaterialTheme.colorScheme.primary
+                    .padding(24.dp)
+                    .fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween
             ) {
-                Icon(Icons.Filled.Edit, contentDescription = "Edit event", tint = MaterialTheme.colorScheme.surface)
+                FloatingActionButton(
+                    onClick = {
+                        FirestoreHelper().deleteEventById(
+                            uid,
+                            parsedDateTime.month.toString(),
+                            parsedDateTime.year.toString(),
+                            eventId
+                        ) {
+                            if (it) {
+                                navController.popBackStack()
+                            } else {
+                                // Handle error
```

Comment:
Didn't really want to pass a context into this, so if there is an error, nothing will happen